Localization: strings, regional formats, tests (#397)
JetBrains/kotlin-multiplatform-dev-docs: topics/compose/compose-localize-strings.md

</resources>
```

Then, create corresponding localized files for translations. For example, add Spanish translations 
to `commonMain/composeResources/values-es/strings.xml`:

```xml
<resources>
    <string name="app_name">Mi Aplicación</string>
    <string name="greeting">¡Hola mundo!</string>
    <string name="welcome_message">¡Bienvenido, %s!</string>
</resources>
```

## Generate class for static access

Once you've added all translations, build the project to generate a special class that provides access to resources.
Compose Multiplatform processes the `strings.xml` resource files in `composeResources` and creates static accessor properties 
for each string resource.

The resulting `Res.strings` object allows you to safely access localized strings from your shared code.
To display strings in your app's UI, use the `stringResource()` composable function. 
This function retrieves the correct text based on the user's current locale:

```kotlin
import project.composeapp.generated.resources.Res

@Composable
fun MyApp() {
    Text(stringResource(Res.strings.app_name))
    Text(stringResource(Res.strings.greeting))
    Text(stringResource(Res.strings.welcome_message, "User"))
}
```

> In the example above, the `welcome_message` string includes a placeholder (`%s`) to insert a dynamic value. 

Reviewer: ```suggestion
In the example above, the `welcome_message` string includes a placeholder (`%s`) for a dynamic value. 
```
